paparazzi-gradle-plugin/src/test/projects/prepare-resources-task-caching/build.gradle.bak

```diff
@@ -0,0 +1,25 @@
+plugins {
+  id 'com.android.library'
+  id 'kotlin-android'
+  id 'app.cash.paparazzi'
+}
+
+android {
+  namespace 'app.cash.paparazzi.plugin.test'
+  compileSdk libs.versions.compileSdk.get() as int
+  defaultConfig {
+    minSdk libs.versions.minSdk.get() as int
+  }
+  compileOptions {
+    sourceCompatibility = libs.versions.javaTarget.get()
+    targetCompatibility = libs.versions.javaTarget.get()
+  }
+  kotlinOptions {
+    jvmTarget = libs.versions.javaTarget.get()
+  }
+  dependencies {
+    implementation "androidx.core:core:1.10.1"
```

Comment:
diff between resources in 1.10.0 and 1.10.1 are 0 so this is a good test of the aar directory change needing to cause the prepare task to run again and not use cache.